Content.Server/Ghost/Roles/Components/GhostRoleComponent.cs

```diff
@@ -67,6 +67,9 @@ public sealed partial class GhostRoleComponent : Component
     [DataField("prob")]
     public float Probability = 1f;
 
+    [DataField("allowPerma")]
```

Comment:
```suggestion
    [DataField]
```
Don't use explicit tags!